neqo-http3/src/server_connection_events.rs

```diff
@@ -38,6 +36,28 @@ pub(crate) struct Http3ServerConnEvents {
     events: Rc<RefCell<VecDeque<Http3ServerConnEvent>>>,
 }
 
+impl RecvMessageEvents for Http3ServerConnEvents {
+    /// Add a new `HeaderReady` event.
+    fn header_ready(&self, stream_id: u64, headers: Option<Vec<Header>>, fin: bool) {
+        self.insert(Http3ServerConnEvent::Headers {
+            stream_id,
+            headers,
+            fin,
+        });
+    }
+
+    /// Add a new `DataReadable` event
+    fn data_readable(&self, stream_id: u64) {
+        self.insert(Http3ServerConnEvent::DataReadable { stream_id });
+    }
+}
+
+impl SendMessageEvents for Http3ServerConnEvents {
+    fn data_writable(&self, _stream_id: u64) {
+        // Curently not used on the server side.
```

Comment:
Do you have an issue open for this?

Reply:
No, but the server side needs a lots of improvements and this is the first think that needs to change. When I am happy with the client side, i will look into the server side.